src/utils/host_file_utils.cc

```diff
@@ -27,16 +28,29 @@
 
 void HostFileUtils::addPuttyKeyToOpenSshKeyFile( const QString& host, const QString& sshhostkeysFileName, const QString& sshKnownHostsFileName )
 {
+#ifdef Q_OS_WIN32
+    // Read the SshHostKey fingerprint form the registry.
+    QString rsaPrefix("0x23,0x");
+    QSettings puttySshHostKey("HKEY_CURRENT_USER\\Software\\SimonTatham\\PUTTY\\SshHostKeys", QSettings::NativeFormat);
+    QString hostKeyValue = puttySshHostKey.value("rsa2@22:kvm-0003.stepping-stone.ch").toString();
+    hostKeyValue.remove(0 , rsaPrefix.length());
+
+    QString sshKey = convertPuttyKey(hostKeyValue, host);
+    addOpenSshKey( sshKey, host, sshKnownHostsFileName);
+#else
 	QString puttyKey = getPuttyKey( host, sshhostkeysFileName );
 	if( puttyKey.isEmpty() )
 	{
 		qDebug() << "no putty key found for host " << host;
 	}
 	else
 	{
+        int prefix = 16 + host.length(); //rsa2@22:host 0x23,0x
+        puttyKey.remove(0,  prefix);
```

Comment:
please use same indentation settings as present code (I know it sucks, but still)